src/services/SplatalogueService.ts

```diff
@@ -125,7 +125,9 @@ export class SplatalogueService {
                     entry = entry?.match(/^(^[\d.]+)/)?.[1] ?? ""; // match the string before the first char that is not a number or "."
                 }
 
-                column.stringData[i] = entry;
+                if (column.stringData !== null && column.stringData !== undefined) {
```

Comment:
We can also use isFinite?

Reply:
It's `string[]`, so we can't use isFinite.